tests/util/test_file.hpp

```diff
@@ -100,7 +103,19 @@ class SyncServer {
 };
 
 struct SyncTestFile : TestFile {
-    SyncTestFile(const realm::SyncConfig&);
+    template<typename BindHandler, typename ErrorHandler>
+    SyncTestFile(const realm::SyncConfig& sync_config, 
+        realm::SyncSessionStopPolicy stop_policy, 
+        BindHandler bind_handler, 
+        ErrorHandler error_handler)
+    {
+        this->sync_config = std::make_shared<realm::SyncConfig>(sync_config);
+        this->sync_config->stop_policy = stop_policy;
+        this->sync_config->bind_session_handler = std::forward<BindHandler>(bind_handler);
```

Comment:
`std::forward` is only meaningful when the argument is declared as `BindHandler&&`